tests/test_loongarch.c

```diff
@@ -0,0 +1,145 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <capstone/capstone.h>
+#include <capstone/platform.h>
+
+struct platform {
+  cs_arch arch;
```

Comment:
please use tabs for indentation, not spaces

Reply:
Thanks, done!